helm-chart/timelord/templates/timelord-ingress.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: {{ .Chart.Name }}-{{ .Release.Name }}-timelord
+  annotations:
+    kubernetes.io/ingress.class: nginx
```

Comment:
I believe the annotation syntax is [deprecated](https://kubernetes.io/docs/concepts/services-networking/ingress/#deprecated-annotation), you should add an `ingressClassName` instead: https://kubernetes.github.io/ingress-nginx/examples/tls-termination/

Reply:
I'll check, but in that case I'd have to make a change to harvester as well.

Reply:
Fixed :+1: